### ISSUE TITLE
deps: linux: Failure while executing: git config --local on CentOS6

### ISSUE THREAD

**Issue by theopolis** [facebook/osquery#4604]
See: https://jenkins.osquery.io/job/osqueryPullRequestLinux/7637/TargetSystem=aws-centos-6/console

```
Error: Failure while executing: git config --local --replace-all homebrew.private false
```

**Comment by theopolis** [comment]
#4595 is attempting to fix this. Hypothesis is the existing CentOS6 build host is in a flaky state.

**Comment by theopolis** [comment]
You need to revert: #4561

**Comment by theopolis** [comment]
Fixed with #4609.

The next step is to move GCC and other bottles in our local formula to be built against the latest Linuxbrew core bottles. Then we can reapply #4561.
